README.md

```diff
@@ -2,196 +2,39 @@
-```
-![](assets/selfedges.png)
-#### Multigraphs
+## Installation
 ```julia
-graphplot([[1,1,2,2],[1,1,1],[1]], names="node_".*string.(1:3), nodeshape=:circle, self_edge_size=0.25)
+]add GraphRecipes
 ```
-![](assets/multigraphs.png)
-#### Arc and chord diagrams
 
+## An example
 ```julia
-using LinearAlgebra
-using SparseArrays
 using GraphRecipes
 using Plots
 
-adjmat = Symmetric(sparse(rand(0:1,8,8)))
-
-plot(
-    graphplot(adjmat,
-              method=:chorddiagram,
-              names=[text(string(i), 8) for i in 1:8],
-              linecolor=:black,
-              fillcolor=:lightgray),
-
-    graphplot(adjmat,
-              method=:arcdiagram,
-              markersize=0.5,
-              linecolor=:black,
-              markercolor=:black)
-    )
-
-```
-![](assets/arc_chord_diagrams.png)
-
-
-#### Julia code -- AST
-
-```julia
-using GraphRecipes
-using Plots
-default(size=(1000, 1000))
-
-code = :(
-function mysum(list)
-    out = 0
-    for value in list
-        out += value
-    end
-    out
-end
-)
-
-plot(code, fontsize=12, shorten=0.01, axis_buffer=0.15, nodeshape=:rect)
-
-```
-
-![](assets/AST_example.png)
-
-#### Julia Type Trees
-
-```julia
-using GraphRecipes
-using Plots
-default(size=(1000, 1000))
-
-plot(AbstractFloat, method=:tree, fontsize=10, nodeshape=:ellipse)
+g = [0 1 1;
+     1 0 1;
+     1 1 0]
 
+graphplot(g,
+          x=[0,-1/tan(π/3),1/tan(π/3)], y=[1,0,0],
+          nodeshape=:circle, nodesize=1.1,
+          axis_buffer=0.6,
+          curves=false,
+          color=:black,
+          nodecolor=[colorant"#389826",colorant"#CB3C33",colorant"#9558B2"],
+          linewidth=10)
 ```
-![](assets/julia_type_tree.png)
-
-
-#### `AbstractTrees` Trees
+![juliagraph](https://user-images.githubusercontent.com/8610352/76132303-27016900-6077-11ea-8fb1-d613c4ddda59.png)
 
-```julia
-using AbstractTrees
-
-AbstractTrees.children(d::Dict) = [p for p in d]
-AbstractTrees.children(p::Pair) = AbstractTrees.children(p[2])
-function AbstractTrees.printnode(io::IO, p::Pair)
-    str = isempty(AbstractTrees.children(p[2])) ? string(p[1], ": ", p[2]) : string(p[1], ": ")
-    print(io, str)
-end
-
-d = Dict(:a => 2,:d => Dict(:b => 4,:c => "Hello"),:e => 5.0)
-
-using GraphRecipes
-using Plots
-default(size=(1000, 1000))
 
-plot(TreePlot(d), method=:tree, fontsize=10, nodeshape=:ellipse)
+### Primary author: Thomas Breloff (@tbreloff)
 
-```
-![](assets/julia_dict_tree.png)
+This repo maintains a collection of recipes for graph analysis, and is a reduced and refactored version of the previous PlotRecipes. It uses the powerful machinery of [Plots](https://github.com/JuliPlots/Plots.jl) and [RecipesBase](https://github.com/JuliaPlots/RecipesBase.jl) to turn simple transformations into flexible visualizations. Recently, the JuliaPlots organization has taken over the maintenance of GraphRecipes, so contacting them (via opening an issue on this repo) will be a faster way to resolve issues than contacting Tom Brellof.
```

Comment:
Just drop the last sentence and the headline above and write `Original author: Thomas Breloff` in normal letters at the very end imho

Reply:
Done

Reply:
Nice. Do you have any position on having your own name on there as primary maintainer? It's not something we've done in the org but maybe we should talk about it

Reply:
Hi @mkborregaard it feels pretty special that you are suggesting that I put my name on the readme!

Having said that, I have thought about this for a bit and I have decided that I am not in favour of having my name on the readme.

I already get the recognition that I need from the JuliaPlots members and from the data that git stores. Furthermore, if people have a problem, then creating issues/asking for help through the usual channels would be more efficient than asking me directly.